wiki/Gameplay/Kiai_time/de.md

```diff
@@ -1,24 +1,23 @@
 ---
-outdated_translation: true
+stub: true
+tags:
+  - timing section
+  - kiai mode
+  - Timing-Sektion
 ---
 
-# Kiai Time
+# Kiai-Zeit
 
-**Kiai Time** wird vom [Mapper](/wiki/Beatmapping) in der [Beatmap](/wiki/Beatmap) zwischen 2 [Timingsektionen](/wiki/Client/Beatmap_editor/Timing) definiert. Sie werden es durch die Sternenfontänen und an dem Geblinke auf Ihrem Bildschirm erkennen, danach werden alle [Hit Objekte](/wiki/Hit_object) im Rhythmus zum Hauptbeat anfangen zu blinken und Sterne werden aus Ihrem Cursor erscheinen, wenn Sie einen [Hit Circle](/wiki/Hit_object/Hit_circle) anklicken oder von Ihrem Cursor runterfallen, wenn Sie die Taste bei einem [Slider](/wiki/Hit_object/Slider) oder einem [Spinner](/wiki/Hit_object/Spinner) durchgehend gedrückt halten.
+*Für Regelungen zur Kiai-Zeit, siehe: [Ranking-Kriterien](/wiki/Ranking_Criteria)*
 
-![Sternenfontänen erscheinen, wenn der Kiai Time beginnt.](img/Kiai_Time_test.jpg "Sternenfontänen erscheinen, wenn der Kiai Time beginnt.")
+**Kiai-Zeit** (oder *Kiai*) ist ein Begriff, der für eine "spezielle" Timing-Sektion genutzt wird, um einen Teil einer [Beatmap](/wiki/Beatmap) besser zu betonen. Spieler können erkennen, ob eine Beatmap eine Kiai-Timing-Sektion hat, wenn sie zusätzliche visuelle Effekte im Hintergrund sehen (z. B. aufleuchtende Beats, fliegende Sterne und Fontänen).
 
-![In der oberen linken Ecke erscheint der Kiai Time, wenn er aktiv ist.](img/Kiai_Time_edit.jpg "In der oberen linken Ecke erscheint der Kiai Time, wenn er aktiv ist.")
+Während Kiai aktiv ist, leuchten alle [Hit-Objekte](/wiki/Hit_object) im Rhythmus des [Haupttakts](/wiki/Client/Beatmap_editor/Timing) auf und Sterne explodieren in der Nähe des Cursors, sobald ein [Hit-Circle](/wiki/Hit_object/Hit_circle) getroffen wurde. Anderweitig fallen sie vom Cursor herunter, wenn ein [Slider](/wiki/Hit_object/Slider) oder [Spinner](/wiki/Hit_object/Spinner) betätigt wurde.
```

Comment:
```suggestion
Während Kiai aktiv ist, leuchten alle [Hit-Objekte](/wiki/Hit_object) im Rhythmus des [Haupttakts](/wiki/Client/Beatmap_editor/Timing) auf und Sterne explodieren in der Nähe des Cursors, sobald ein [Hit-Circle](/wiki/Hit_object/Hit_circle) getroffen wurde. Anderweitig fallen sie vom Cursor herunter, wenn ein [Slider](/wiki/Hit_object/Slider) geschoben oder ein [Spinner](/wiki/Hit_object/Spinner) gedreht wurde.
```

Reply:
Could also do `Slider gehalten`.

Reply:
Editing in the other suggestion, due to the two being in the same line.